inc/shortcodes/class-pdf.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Shortcake_Bakery\Shortcodes;
+
+class PDF extends Shortcode {
+
+	public static function get_shortcode_ui_args() {
+		return array(
+			'label'          => esc_html__( 'PDF', 'shortcake-bakery' ),
+			'listItemImage'  => 'dashicons-media-document',
+			'attrs'          => array(
+				array(
+					'label'  => esc_html__( 'URL', 'shortcake-bakery' ),
+					'attr'   => 'url',
+					'type'   => 'text',
+				),
+			),
+		);
+	}
+
+	/**
+	 * Turn embed code into a proper shortcode
+	 *
+	 * @param  string $content
+	 * @return string $content
+	 */
+	public static function reversal( $content ) {
+		$needle = '#http[^<]//[^<]+\.pdf?#';
```

Comment:
@goldenapples @danielbachhuber Updated the regex to only match pattern of URL in between two blank lines. This will miss the edge case of PDFs in the first line, but not sure of a better way to solve that.